cli/lib/build.js

```diff
@@ -52,6 +52,54 @@ const checkBranch = async () => {
   }
 };
 
+const checkCodeUpToDate = async () => {
+  logInfo('update code');
+
+  let answer = await inquirer.prompt([
+    {
+      type: 'confirm',
+      name: 'updateCode',
+      message: 'Whether to auto update the code? Or you can update manually and then select "N" to continue.',
+      default: false,
+    },
+  ]);
+
+  if (answer.updateCode) {
+    logInfo(`performing "git pull upstream develop" inside "${rootDir}"`);
+    await execSync(`git pull upstream develop`, { cwd: rootDir });
```

Comment:
not every has the default upstream name as `upstream`, e.g. I rename it as `erda-source`, then if I run this script, I would get error

Reply:
![image](https://user-images.githubusercontent.com/5175455/117390872-7d8b6100-af21-11eb-9578-8d1905015e14.png)